camshare/camshare.xcworkspace/xcuserdata/jerasmus.xcuserdatad/xcdebugger/Breakpoints_v2.xcbkptlist

```diff
@@ -8,7 +8,7 @@
          BreakpointExtensionID = "Xcode.Breakpoint.FileBreakpoint">
```

Comment:
Avoid uploading xcuserdata.

Reply:
This file is purely for Breakpoints though, we should avoid adding this as Brandon mentioned

Reply:
Cool thanks Zaheer, Brandon explained on slack